tests/test_build.py

```diff
@@ -23,3 +23,32 @@ def test_run_pymodules_install_optional_project_dir(self):
             assert run_pymodules_install(ctx, modules, project_dir) is None
         assert m_info.call_args_list[-1] == mock.call(
             'No Python modules and no setup.py to process, skipping')
+
+    def test_strip_if_debuggable(self):
+        ctx = mock.Mock()
+        ctx.python_recipe.major_minor_version_string = "python3.6"
+        ctx.get_site_packages_dir.return_value = "test-doesntexist"
+        ctx.build_dir = "nonexistant_directory"
+        ctx.archs = ["arm64"]
+
+        modules = ["mymodule"]
+        project_dir = None
+        with mock.patch('pythonforandroid.build.info'), \
+                mock.patch('sh.Command'),\
+                mock.patch('pythonforandroid.build.open'),\
+                mock.patch('pythonforandroid.build.shprint'),\
+                mock.patch('pythonforandroid.build.current_directory'),\
+                mock.patch('pythonforandroid.build.CythonRecipe') as m_CythonRecipe, \
+                mock.patch('pythonforandroid.build.project_has_setup_py') as m_project_has_setup_py, \
+                mock.patch('pythonforandroid.build.run_setuppy_install'):
+            m_project_has_setup_py.return_value = False
+
+            # Make sure it is NOT called when debug build:
+            ctx.build_as_debuggable = True
+            assert run_pymodules_install(ctx, modules, project_dir) is None
+            self.assertFalse(m_CythonRecipe().strip_object_files.called)
```

Comment:
Minor: why don't we make it consistent via the `assert` keyword?
```python
assert m_CythonRecipe().strip_object_files.called is False
```
Note how I purposely didn't write:
```python
assert not m_CythonRecipe().strip_object_files.called
```
Because I also want to enforce type checking, specially for mocks and unit testing